libs/docs/src/docs/no-version/creating-a-devfile-stack.md

```diff
@@ -9,16 +9,161 @@ various projects.
+        env:
+            - name: CONFIGURATION
+            value: Debug
+            - name: STARTUP_PROJECT
+            value: app.csproj
+            - name: ASPNETCORE_ENVIRONMENT
+            value: Development
+            - name: ASPNETCORE_URLS
+            value: http://*:8080
+        endpoints:
+            - name: http-dotnet60
+            targetPort: 8080
+    ```
+
+9. Finally, add the `Commands` inside your `devfile.yaml`:
+    ```yaml {% filename="devfile.yaml" %}
+    commands:
+    - id: build
+        exec:
+        workingDir: ${PROJECT_SOURCE}
+        commandLine: kill $(pidof dotnet); dotnet build -c $CONFIGURATION $STARTUP_PROJECT /p:UseSharedCompilation=false
+        component: dotnet
+        group:
+            isDefault: true
+            kind: build
+    - id: run
+        exec:
+        workingDir: ${PROJECT_SOURCE}
+        commandLine: dotnet run -c $CONFIGURATION --no-build --project $STARTUP_PROJECT --no-launch-profile
+        component: dotnet
+        group:
+            isDefault: true
+            kind: run
+    ```
+
+10. The full `devfile.yaml` will be:
+    ```yaml {% filename="devfile.yaml" %}
+    schemaVersion: 2.1.0
+    metadata:
+    name: tutorial-stack
+    displayName: Tutorial Stack
+    description: Test stack based on .Net 6.0
+    icon: https://github.com/dotnet/brand/raw/main/logo/dotnet-logo.png
+    tags:
+        - .NET
+    projectType: dotnet
+    language: .NET
+    version: 1.0.3
+    starterProjects:
+    - name: tutorial-example
+        git:
+        checkoutFrom:
+            remote: origin
+            revision: dotnet-6.0
+        remotes:
+            origin: https://github.com/redhat-developer/s2i-dotnetcore-ex
+        subDir: app
+    components:
+    - name: dotnet
+        container:
+        image: registry.access.redhat.com/ubi8/dotnet-60:6.0
+        args: ["tail", "-f", "/dev/null"]
+        mountSources: true
+        env:
+            - name: CONFIGURATION
+            value: Debug
+            - name: STARTUP_PROJECT
+            value: app.csproj
+            - name: ASPNETCORE_ENVIRONMENT
+            value: Development
+            - name: ASPNETCORE_URLS
+            value: http://*:8080
+        endpoints:
+            - name: http-dotnet60
+            targetPort: 8080
+    commands:
+    - id: build
+        exec:
+        workingDir: ${PROJECT_SOURCE}
+        commandLine: kill $(pidof dotnet); dotnet build -c $CONFIGURATION $STARTUP_PROJECT /p:UseSharedCompilation=false
+        component: dotnet
+        group:
+            isDefault: true
+            kind: build
+    - id: run
+        exec:
+        workingDir: ${PROJECT_SOURCE}
+        commandLine: dotnet run -c $CONFIGURATION --no-build --project $STARTUP_PROJECT --no-launch-profile
+        component: dotnet
+        group:
+            isDefault: true
+            kind: run
+
+    ```
+
+11. Create an Pull Request from your fork to the [devfile-registry](https://github.com/devfile/registry).
```

Comment:
```suggestion
11. Create a Pull Request from your fork to the [devfile-registry](https://github.com/devfile/registry).
```

Reply:
I think this also must be at the end. I've added a stack.yaml example too :)